benchmarks/bulk-insert-and-query.cc

```diff
@@ -394,28 +391,12 @@ int main(int argc, char * argv[]) {
 
     {80, "Morton"},
 
-    {96, "XorBinaryFuse8-Naive"},
-    {97, "XorBinaryFuse16-Naive"},
-    {98, "XorBinaryFuse8-4Wise-Prefetch"},
-    {99, "XorBinaryFuse16-4Wise-Prefetch"},
-    {100, "XorBinaryFuse8-Prefetched"},
-    {101, "XorBinaryFuse16-Prefetched"},
-    {102, "XorBinaryFuse8-4Wise"},
-    {103, "XorBinaryFuse16-4Wise"},
-    {106, "XorBinaryFuse8-Sorted"},
-    {107, "XorBinaryFuse16-Sorted"},
-    {108, "XorBinaryFuse8-PSorted"},
-    {109, "XorBinaryFuse16-PSorted"},
-    {110, "XorBinaryFuse8-FSorted"},
-    {111, "XorBinaryFuse16-FSorted"},
-    {112, "XorBinaryFuse8-4Wise-PSorted"},
-    {113, "XorBinaryFuse16-4Wise-PSorted"},
-    {114, "XorBinaryFuse8-OneHash"},
-    {115, "XorBinaryFuse16-OneHash"},
-    {116, "XorBinaryFuse8"},
-    {117, "XorBinaryFuse16"},
-    {118, "XorBinaryFuse8-4Wise"},
-    {119, "XorBinaryFuse16-4Wise"},
+    {96, "XorBinaryFuse8 (Naive)"},
+    {97, "XorBinaryFuse16 (Naive)"},
+    {116, "XorBinaryFuse8-LowMem"},
```

Comment:
Not sure why you want to use the "LowMem" term here... sure it uses a bit less memory, but not a lot. Why use an extension at all? Isn't "XorBinaryFuse8" sufficient?